mrp_brewing/models/stock.py

```diff
@@ -27,15 +27,24 @@ class StockProductionLot(models.Model):
     _inherit = 'stock.production.lot'
     
     qty_available = fields.Float(compute="_compute_qty_available", string='Quantity available',store=True)
-    
+
+    def _compute_lot_quantity(self, lot):

```

Comment:
as far as you give the lot to the function you can also assign qty_available to lot.qty_available

lot.qty_available = qty_available should be in the function IMHO

Reply:
Yup, I did that at first then I changed it this way to avoid side effects within the function : functional reflexes.

I'll follow your intuition on this one. :-)